src/py/flwr/cli/log.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower command line interface `log` command."""
+
+import time
+from logging import DEBUG, INFO
+
+import grpc
+import typer
+from typing_extensions import Annotated
+
+from flwr.common.grpc import GRPC_MAX_MESSAGE_LENGTH, create_channel
+from flwr.common.logger import log as logger
+
+STREAM_DURATION: int = 60
```

Comment:
Is this related to the frequency at which the logs are streamed/pulled (i.e. every this many seconds)? if so maybe we should rename it?

Reply:
@jafermarq Yes! It is related, though I'd say it's a matter of perspective. 

Duration (or period) being the allowable elapsed time, whereas frequency is the inverse of it (once every `int` elapsed time). I favour `duration`/`period` or something like that vs `frequency` which to me reads the units of `Hz`, which in the example above would say 60 times per second.

Maybe I'm overthinking it, lmk @jafermarq.

Reply:
I prefer "`period`" over "`duration`". With "`duration`" it gives the impression the logs are streamed for that many seconds and then it stops. Refreshing after 60s might not be frequent enough? Should we set it as an input arg right from the start?

Reply:
Sure, I'll change it to `period`. From my tests, 60s might be too short - it depends on how fast clients train and send replies to the ServerApp. I'd say, use 60s as the default and users can set a different duration if required. Wdyt @jafermarq?